projects/core/library/ansible_toolbox.py

```diff
@@ -262,6 +262,34 @@ def _run(self):
 
                 host_properties.append("ansible_user="+remote_username)
 
+            # Configure OS-specific Ansible variables
+            remote_os = env.get("TOPSAIL_REMOTE_OS")
+            os_configurations = {
```

Comment:
can you move this outside of the function? like at the top of the file, wither as Python object, or with something like (I find yaml more readable than anything else)
```
ANSIBLE_OS_CONFIGS = """
windows:
  properties: ...
```
and when you need it: `ansible_os_configs = yaml.safe_load(ANSIBLE_OS_CONFIGS)`